### ISSUE TITLE
somewhat misleading startup message mentioning localhost

### ISSUE THREAD

**Issue by ldemailly** [fortio/fortio#110]
When fortio is started in server mode it currently prints:
```
$ fortio server
UI starting - visit:
http://localhost:8080/fortio/
Https redirector running on :8081
Fortio 0.6.7 grpc ping server listening on port 8079
Fortio 0.6.7 echo server listening on port 8080
```

The mention of localhost is for the convenience of copy-pasting the url on the local machine but may lead people to believe it's only binding on localhost which isn't the case (like the other 3 messages show, it's just binding on the 3 ports, on all interfaces)

related to #91 

cc @danehans  who pointed out the issue - thanks!

**Comment by danehans** [comment]
@ldemailly I believe https://github.com/istio/fortio/pull/109 closes out this issue. When I specify `ip:port` for the `-http-port` flag, Fortio prints the expected output:
```
UI starting - visit:
http://$IP:$PORT/fortio/
```
When I specify a `port` for `http-port`, Fortio prints the expected `localhost:$PORT` output:
```
UI starting - visit:
http://localhost:$PORT/fortio/
```

Please see the output below of my testing:
```
$ docker run -p 8088:8088 -p 8076:8076 istio/fortio server -http-port 172.17.0.2:8088 -grpc-port 8076 -loglevel verbose
21:00:16 I logger.go:97> Log level is now 1 Verbose (was 2 Info)
21:00:16 V stats.go:491> Will use [50 75 99 99.9] for percentiles
21:00:16 I uihandler.go:91> Using resources directory set at link time: /usr/local/lib/fortio
Https redirector running on :8081
Fortio 0.6.8 grpc ping server listening on port 8076
UI starting - visit:
http://172.17.0.2:8088/fortio/
Fortio 0.6.8 echo server listening on port 172.17.0.2:8088

$ docker run -p 8088:8088 -p 8076:8076 istio/fortio server -http-port 8088 -grpc-port 8076 -loglevel verbose
21:01:12 I logger.go:97> Log level is now 1 Verbose (was 2 Info)
21:01:12 V stats.go:491> Will use [50 75 99 99.9] for percentiles
21:01:12 I uihandler.go:91> Using resources directory set at link time: /usr/local/lib/fortio
Fortio 0.6.8 grpc ping server listening on port 8076
Https redirector running on :8081
UI starting - visit:
http://localhost:8088/fortio/
Fortio 0.6.8 echo server listening on port :8088
```

**Comment by ldemailly** [comment]
I still ideally want the default to show localhost for convenience but indicate we are actually listening on all addresses

eg like I put in https://github.com/istio/fortio/pull/109#issuecomment-363529294

show
```
UI starting - visit:
http://localhost:8080/fortio/   (or any host/ip reaching this machine)
```

when not specifying a bind address

**Comment by danehans** [comment]
That is what I currently see:
```
$ docker run -p 8088:8088 -p 8076:8076 istio/fortio server -http-port 8088 -grpc-port 8076 -loglevel verbose
21:01:12 I logger.go:97> Log level is now 1 Verbose (was 2 Info)
21:01:12 V stats.go:491> Will use [50 75 99 99.9] for percentiles
21:01:12 I uihandler.go:91> Using resources directory set at link time: /usr/local/lib/fortio
Fortio 0.6.8 grpc ping server listening on port 8076
Https redirector running on :8081
UI starting - visit:
http://localhost:8088/fortio/
Fortio 0.6.8 echo server listening on port :8088
```

**Comment by ldemailly** [comment]
yes so cosmetically, in that no address provided case, it would be good to have it say
```
UI starting - visit:
http://localhost:8088/fortio/   (or any host/ip reaching this machine)
```
instead of
```
UI starting - visit:
http://localhost:8088/fortio/
```

I am open to better suggestion for the text I suggest to add `(or any host/ip reaching this machine)`

**Comment by danehans** [comment]
Duh... now I see what you're saying. Let me take a crack at it.

**Comment by danehans** [comment]
Looks like I have it now.

__Test 1:__ No `ip:port` for `-http-port` flag provides `(or any host/ip reaching this machine)` message:
```
$ docker run --name fortio --rm -p 8088:8088 -p 8078:8078 istio/fortio:webtest server -loglevel verbose -http-port 8088 -grpc-port 8078
20:05:05 I logger.go:97> Log level is now 1 Verbose (was 2 Info)
20:05:05 V stats.go:491> Will use [50 75 99 99.9] for percentiles
Https redirector running on :8081
UI starting - visit:
http://localhost:8088/fortio/   (or any host/ip reaching this machine)
Fortio 0.6.8 grpc ping server listening on port 8078
Fortio 0.6.8 echo server listening on port :8088
```

__Test 2:__ Specify an `ip:port` for `-http-port` flag provides __no__`(or any host/ip reaching this machine)` message.
```
$ docker run --name fortio --rm -p 8088:8088 -p 8079:8079 istio/fortio:webtest server -loglevel verbose -http-port 172.17.0.2:8088
20:07:16 I logger.go:97> Log level is now 1 Verbose (was 2 Info)
20:07:16 V stats.go:491> Will use [50 75 99 99.9] for percentiles
20:07:16 I uihandler.go:91> Using resources directory set at link time: /usr/local/lib/fortio
Fortio 0.6.8 grpc ping server listening on port 8079
UI starting - visit:
http://172.17.0.2:8088/fortio/
Https redirector running on :8081
Fortio 0.6.8 echo server listening on port 172.17.0.2:8088
```

**Comment by ldemailly** [comment]
this got broken (cosmetically) in #155 I should have tested myself a bit

**Comment by olimpias** [comment]
Are you working on this? Or should I fix?

**Comment by ldemailly** [comment]
> Are you working on this? Or should I fix?

I asked you the same question :-)  I'll fix it I guess, while you finish #83
